copier/copier.go

```diff
@@ -1785,3 +1831,22 @@ func copierHandlerMkdir(req request, idMappings *idtools.IDMappings) (*response,
 
 	return &response{Error: "", Mkdir: mkdirResponse{}}, nil, nil
 }
+
+func copierHandlerRemove(req request) *response {
+	errorResponse := func(fmtspec string, args ...interface{}) *response {
+		return &response{Error: fmt.Sprintf(fmtspec, args...), Remove: removeResponse{}}
+	}
+	resolvedTarget, err := resolvePath(req.Root, req.Directory, false, nil)
+	if err != nil {
+		return errorResponse("copier: remove: error resolving %q: %v", req.Directory, err)
```

Comment:
This is another stutter.  resolvePath reports the directory.

Reply:
Replacing the format string with `copier: remove: %v`.